R/Reporter.R

```diff
@@ -57,7 +57,7 @@ Reporter <- R6::R6Class( # nolint: object_name_linter.
     #' @description Retrieves all `ReportCard` objects contained in the `Reporter`.
     #'
     #' @return A (`list`) of [`ReportCard`] objects.
-    #' @examples
+    #' @examplesIf require("ggplot2")
```

Comment:
@donyunardi in other packages (like `teal.slice`) we have `requireNamespace` in @examplesIf

```suggestion
    #' @examplesIf requireNamespace("ggplot2")
```

Reply:
Alrighty, so I think we should update other packages where it got `requireNamespace`